Modelica/Electrical/PowerConverters.mo

```diff
@@ -90,6 +92,41 @@ is provided.
 </html>"));
     end DCDCConcept;
 
+    class ACACConcept "AC/AC converter concept"
+      extends Modelica.Icons.Information;
+      annotation (DocumentationClass=true, Documentation(info="<html>
+
+<p>The following DC/DC converter topologies are currently included in the PowerConverters library.</p>
+
+<ul>
+<li>Single-phase dimmer with <a href=\"modelica://Modelica.Electrical.PowerConverters.ACAC.SinglePhaseTriac\">triac</a></li>
+<li>Polyphase induction machine soft starter with <a href=\"modelica://Modelica.Electrical.PowerConverters.ACAC.PolyPhaseTriac\">triac</a></li>
+</ul>
+
+<h4>Control</h4>
+
+<p>To apply firing signals to the triac, the 
+<a href=\"modelica://Modelica.Electrical.PowerConverters.DCDC.Control.Signal2mPulse\">Signal2mPulse adaptor</a> is provided.
+</p>
+<p>
+The <a href=\"modelica://Modelica.Electrical.PowerConverters.ACAC.Control.VoltageToAngle\">Voltage2Angle block</a> 
+calculates phase angle from reference voltage.
+</p>
+<p>
+To control the soft start of an induction machine, 
+the <a href=\"modelica://Modelica.Electrical.PowerConverters.ACAC.Control.SoftStartControl\">SoftStartControl block</a> 
+is provided. It applies a voltage ramp during start, setting the ramp on hold whenever the measured current exceeds the maximum current. 
+Furthermore, a ramp down can be applied for stoppng the drive.
```

Comment:
```suggestion
Furthermore, a ramp down can be applied for stopping the drive.
```

Reply:
GitHub claims that I have no permission to push to this repository... 

![image](https://user-images.githubusercontent.com/4184218/59835339-c0ea7f80-9349-11e9-83c4-cdc4757a6b92.png)

So I klick on `Resolve conversation`. I hope this accepts the proposed changes

Reply:
No, the resolved conversation seems to imply that I have to resolve the issue manually. OK.

Reply:
Only the creator of the PR (@AHaumer) can accept those code suggestions.

Reply:
OK. I then way for the push of @AHaumer.

Reply:
done